src/loadsave.jl

```diff
@@ -159,3 +159,51 @@ function loadtodict!(d::Dict, g::Union{JLDFile,Group}, prefix::String="")
     end
     return d
 end
+
+"""
+    save_object(filename, x)
+
+Write one object `x` from the current scope to a JLD2 file `filename`. The
+object is stored with a default name of "single_stored_object". If the default
+name needs to be overridden use [`@save`](@ref) instead. If the file exists, it
+overwrites it.
+
+# Example
+
+To save the string `hello` to the JLD2 file example.jld2:
+
+    hello = "world"
+    save_object("example.jld2", hello)
+"""
+function save_object(filename, x)
+  jldopen(filename, "w") do file
+    file["single_stored_object"] = x
+  end
+  return
+end
+
+"""
+    load_object(filename)
+
+Load the only available object from the JLD2 file `filename`. Provides
+functionality similar to `loadRDS`. If `filename` contains more than one object,
+the function throws an `ArgumentError`. The object name is not loaded into namespace. For
+loading more than one object or objects directly into namespace, use
+[`@load`](@ref).
```

Comment:
```suggestion
Load and returns the only available object from the JLD2 file `filename`. 
If the file contains more than one object, the function throws an `ArgumentError`.

For loading more than one object, use [`jldopen`](@ref), the FileIO API or [`@load`](@ref).
```
I've rephrased this docstring to account for the following points
 * many readers won't have a clue what `loadRDS` is 
 * I think not writing stuff into global scope can safely be assumed to be the default behaviour of any well-behaved function (at least in Julia) and shouldn't be mentioned explicitely
 * as above, there are more ways to handle multiple objects (and I personally dislike `@load`)

Reply:
Personally, I like `@save` and `@load`. That aside I have incorporated your suggested changes in the latest commit.

Reply:
I'll admit that my asides about `@save` and `@load` are completely unrelated to the issue at hand and slightly unprofessional.